src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateHiddenMarkovModel.java

```diff
@@ -1,89 +1,87 @@
 package org.broadinstitute.hellbender.tools.exome.germlinehmm;
 
-import com.google.common.annotations.VisibleForTesting;
-import org.apache.commons.math3.random.RandomGenerator;
-import org.apache.commons.math3.random.RandomGeneratorFactory;
-import org.broadinstitute.hellbender.tools.coveragemodel.XHMMTargetLikelihoodCalculator;
+import org.broadinstitute.hellbender.tools.coveragemodel.TargetLikelihoodCalculator;
+import org.broadinstitute.hellbender.tools.coveragemodel.XHMMEmissionProbabilityCalculator;
 import org.broadinstitute.hellbender.tools.exome.Target;
-import org.broadinstitute.hellbender.utils.SimpleInterval;
 import org.broadinstitute.hellbender.utils.Utils;
-import org.broadinstitute.hellbender.utils.hmm.CopyNumberTriState;
-import org.broadinstitute.hellbender.utils.hmm.CopyNumberTriStateTransitionProbabilityCache;
 import org.broadinstitute.hellbender.utils.hmm.HiddenMarkovModel;
 import org.broadinstitute.hellbender.utils.param.ParamUtils;
 
+import javax.annotation.Nonnull;
+import java.io.Serializable;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.Random;
 
 /**
- *  This model has the following parameters:
- *  Event start probability: the probability of a CNV event beginning at any given base.
- *  Mean event size: the average size (in base-pairs) of an event.
- *  Mean coverage shift for deletions: the mean coverage in regions that have undergone a copy loss.
- *  Mean coverage shift for duplications:  the mean coverage in regions that have undergone a copy gain.
+ *  A generic copy number tri-state HMM has the following parameters:
```

Comment:
I personally like HTML tags, though, I tend to use them only when I have a few minutes to kill! :)